src/main/resources/assets/appliedenergistics2/lang/de_de.lang

```diff
@@ -390,6 +442,10 @@ item.appliedenergistics2.material.cell1k_part.name=1k-ME-Speicherkomponente
 item.appliedenergistics2.material.cell4k_part.name=4k-ME-Speicherkomponente
 item.appliedenergistics2.material.cell16k_part.name=16k-ME-Speicherkomponente
 item.appliedenergistics2.material.cell64k_part.name=64k-ME-Speicherkomponente
+item.appliedenergistics2.material.fluid_cell1k_part.name=1k-ME-Flüssigkeitenkomponente
+item.appliedenergistics2.material.fluid_cell4k_part.name=4k-ME-Flüssigkeitenkomponente
+item.appliedenergistics2.material.fluid_cell16k_part.name=16k-ME-Flüssigkeitenkomponente
+item.appliedenergistics2.material.fluid_cell64k_part.name=64k-ME-Flüssigkeitenkomponente
```

Comment:
To make the translation a bit more clear, we should say "Flüssigkeitenspeicherkomponente" instead of "Flüssigkeitenkomponente".

Reply:
```suggestion
item.appliedenergistics2.material.fluid_cell1k_part.name=1k-ME-Flüssigkeitenspeicherkomponente
item.appliedenergistics2.material.fluid_cell4k_part.name=4k-ME-Flüssigkeitenspeicherkomponente
item.appliedenergistics2.material.fluid_cell16k_part.name=16k-ME-Flüssigkeitenspeicherkomponente
item.appliedenergistics2.material.fluid_cell64k_part.name=64k-ME-Flüssigkeitenspeicherkomponente
```